Models/CLEM/Resources/Ruminant.cs

```diff
@@ -83,30 +85,19 @@ public RuminantFemale Mother
             {
                 mother = value;
                 if (mother != null)
-                {
                     MotherID = value.ID;
-                }
             }
         }
+
         /// <summary>
         /// Link to individual's mother
         /// </summary>
         public int MotherID { get; private set; }
 
-        /// <summary>
-        /// Gender
-        /// </summary>
-        public Sex Gender { get; set; }
-
         /// <summary>
         /// Sex of individual
         /// </summary>
-        public Sex Sex { get { return (this is RuminantFemale)?Sex.Female:Sex.Male; } }
-
-        /// <summary>
-        /// Gender as string for reports
-        /// </summary>
-        public string GenderAsString { get { return Gender.ToString().Substring(0,1); } }
+        public virtual Sex Sex { get { return this.Sex; } }
```

Comment:
```suggestion
        public abstract Sex Sex { get; }
```

Then in the derived types:

```csharp
public override Sex Sex { get { return ... } }
```